support-cycle.md

```diff
@@ -0,0 +1,180 @@
+
+##### Validating Safari support
+
+We validate Safari support by running Continuous Integration testing for the oldest supported version of iOS and MacOS, as well as the latest supported versions of both. Any versions inbetween are not tested as it is unlikely that there will be failures in specific versions between the oldest supported version and the latest release that aren't reflected in one or the other.
+
+##### What does this look like?
+
+The following predicts our support cycle until 2020. This is an estimate and may change based on the Safari release cycle.
+
+ - We will always have hard support for the latest version
+ - Safari 8 will receive support until around 2017-10-01
+ - Safari 9 will receive support until around 2018-10-01
+ - Safari 10 will receive support until around 2019-10-01
+ - Safari 11 will receive support until around 2020-10-01
+
+#### Edge
+
+Microsoft have no documentation of the Microsoft Edge Support Cycle. Public releases tend to be made around 6±2 months. We currently support the latest release of Microsoft Edge, with soft-support for public releases up to a year old (that is, we do not test older releases, but if a user finds a bug we will fix it).
+
+##### Validating Edge support
+
+We validate support of Edge by running Continuous Integration testing for the latest version. Older versions aren't tested due to build time constraints.
+
+##### What does this look like?
+
+The following predicts our support cycle until 2020. This is an estimate and may change based on the Safari release cycle.
+
+ - We will always have hard support for the latest version
+ - Edge 15 will receive soft-support until around 2018-04-01
+ - Edge 16 will receive soft-support until around 2018-10-01
+ - Edge 17 will receive soft-support until around 2019-04-01
+ - Edge 18 will receive soft-support until around 2019-10-01
+ - Edge 19 will receive soft-support until around 2020-04-01
+
+#### Internet Explorer
+
+[Microsoft have officially stopped support for Internet Explorer 10 and below since January 2016](https://www.microsoft.com/en-gb/windowsforbusiness/end-of-ie-support), but still continue to support Internet Explorer 11 in Windows 7, 8 and 10. The end-of-life date for Windows 10 is 2025-10-14 as explained in the [Microsoft LifeCycle Fact sheet](https://support.microsoft.com/en-us/help/13853/windows-lifecycle-fact-sheet), and so we can expect to support Internet Explorer 11 until then. We do not support older versions of Internet Explorer.
```

Comment:
So is our official stance to no longer support IE 10? Mostly thinking about this: https://github.com/chaijs/chai/issues/860

Reply:
I don't have hugely strong opinions on whether we keep supporting IE10. I'm happy to drop it, I'm happy to keep supporting it. Mostly, decisions in this initial draft were guided by vendor support and not set in stone; the intent was to document produce _some decision_ which can trigger a conversation and we can settle on something else should we need to (to which I guess this has served it's purpose). 

To try and input into a discussion of supporting IE10, let's state some facts:

 - This document encodes that Chai 4 will continue to support IE 10
 - This document encodes that all Chai projects will support IE 11 until 2025, it's EOL date.

Now some questions: Should we just not change this section, and have it as an implicit guide that we drop IE10 for Chai 5? Conversely, would it be feasible to _support_ IE10 _and_ IE11 to 2025? These I think are the two extremes we have to chose from, is there a sensible non-arbitrary choice in between those?

Reply:
I think we can simply drop IE 10, can't we?

I remember we have had some discussion around this in this issue (https://github.com/chaijs/chai/issues/860) and considering what has been said there it would be better to drop it.